src/docs/tutorials/04-edit-email-templates-with-flex-cli/index.md

```diff
@@ -188,19 +188,88 @@ text editor and make a simple change:
 
 When you've made the change, save the file.
 
+## Preview your changes
+
+You can test your changes and templates by previewing or sending a test
+email using your local template files. To preview the change above:
+
+```bash
+flex-cli notifications preview -m my-test-marketplace --template preauth-nightly-booking/templates/new-booking-request
```

Comment:
Only catching & commenting this now but I feel the parameter order would be more logical if the`-m my-test-marketplace` comes as a the very last parameter for the command because it's not a command specific parameter but a global one, and later we might make it go away by supporting something like global set marketplace. This of course should be consistent at least within this doc.